docs/python/1.6/python-appapi-devguide.md

```diff
@@ -12,16 +12,19 @@ next:
 
 Python is a popular language with a large and comprehensive standard library as well as many [third-party libraries](https://pypi.python.org). The IBM Streams Python Application API enables you to create streaming analytics applications in Python.
 
-The API is open source from the [streamsx.topology](http://ibmstreams.github.io/streamsx.topology/) project on GitHub. IBM Streams 4.2 ships with a release of `com.ibm.streamsx.topology` toolkit which contains the Python Application API.
+The API is open source from the [streamsx.topology](http://ibmstreams.github.io/streamsx.topology/) project on GitHub. IBM Streams 4.2.1 ships with a release of `com.ibm.streamsx.topology` toolkit which contains version 1.6 the Python Application API.
 
 This guide covers the high-level concepts of streaming application development with Python. The guide also walks you through the process of creating a sample application to help you get more familiar with how to create an IBM Streams application with Python.
 
 When you create an IBM Streams application written in Python, you can run the application in the following modes:
 
-* As a **Streams distributed application** (DISTRIBUTED). When running in this mode, the application produced will be deployed automatically on your IBM Streams instance.
-* As a **Streams Application Bundle file** (BUNDLE). When running in this mode, the application produces a SAB file that you can then deploy on your IBM Streams or Bluemix Streaming Analytics service instance by using the `streamtool submitjob` command or by using the application console
+* As a **Streams Application Bundle file** (BUNDLE). When running in this mode, the application produces a Streams Application Bundle (SAB) file that you can then deploy on your IBM Streams or Bluemix Streaming Analytics service instance by using the `streamtool submitjob` command or by using the application console
 * As a **stand-alone application** (STANDALONE).  When running in this mode, the application produces a Streams Application Bundle file (SAB file), but rather than submitting the SAB file to an instance, the bundle is executed. The bundle runs within a single process and can be terminated with Ctrl-C interrupts.
+* As a **Streams distributed application** (DISTRIBUTED). When running in this mode, the application produced will be deployed automatically on your IBM Streams instance.
+* As a **Bluemix Streaming Analytics Application** (ANALYTICS_SERVICE). This mode the application will be run in the cloud in a Bluemix Streaming Analytics Service.
```

Comment:
@wmarshall484 Dan confirmed that our docs should use STREAMING_ANALYTICS_SERVICE going forward. That term avoids ambiguity. However, ANALYTICS_SERVICE will still work and is documented in the Ref.  Let me know when the final changes are done so I can merge the pull request.